wfl/Models/RestaurantDto.cs

```diff
@@ -1,13 +1,11 @@
-﻿using MongoDB.Bson.Serialization.IdGenerators;
-using MongoDB.Bson.Serialization.Attributes;
+﻿﻿using System;
 
 namespace Symphono.Wfl.Models
 {
     public class RestaurantDto
     {
-        [BsonIgnoreIfNull]
-        [BsonId(IdGenerator = typeof(StringObjectIdGenerator))]
         public string Id { get; set; }
         public string Name { get; set; }
+        public Uri MenuLink { get; set; }
```

Comment:
What does this actually end up saving in mongo?? When saving does it just `.ToString()`?